src/cf-typography/src/atoms/links.less

```diff
@@ -126,16 +54,17 @@
             left: 0;
         }
```

Comment:
Same comment as above, this is necessary for the icon to be placed appropriately inside a block link